doc/source/cluster/key-concepts.rst

```diff
@@ -60,19 +60,31 @@ sit idle, the autoscaler will remove worker nodes from the cluster.
 It is important to understand that the autoscaler only reacts to task and actor resource requests, and not application metrics or physical resource utilization.
 To learn more about autoscaling, refer to the user guides for Ray clusters on :ref:`VMs <cloud-vm-index>` and :ref:`Kubernetes <kuberay-index>`.
 
+
+.. _ray_driver:
+
+Ray Driver
```

Comment:
Why is this not a part of "running_applications" section? I feel like the right structure is

1. Key concept explains this
2. Under running applications, we should demonstrate (with example) 3 ways to run ray workloads to the ray cluster. 1. driver. 2. job submission. 3. ray client?

Reply:
Addressed, defined driver inline